packages/linter/package.json

```diff
@@ -46,8 +46,9 @@
     "clean": "touch lib && rm -rf lib && mkdir lib",
     "format": "find src | grep \"\\.[jt]s$\" | xargs clang-format --style=file -i",
     "lint": "tslint -c tslint.json src/*.ts src/**/*.ts",
-    "test": "npm run build && mocha && npm run lint",
-    "test:watch": "tsc-then -- mocha",
+    "test": "npm run build && npm run test:unit && npm run lint",
+    "test:unit": "mocha",
+    "test:watch": "tsc-then -- npm run test:unit",
```

Comment:
prefer just tsc-then -- mocha

Reply:
I've changed all `npm run test:unit` inside of npm scripts to `mocha` where appropriate.